modules/objdetect/test/test_charucodetection.cpp

```diff
@@ -689,6 +689,25 @@ TEST(Charuco, testmatchImagePoints)
     }
 }
 
+TEST(Charuco, testDrawDetected) {
+    vector<vector<Point2f>> detected_float = {{Point2f(0.f, 0.f), Point2f(10.f, 0.f), Point2f(10.f, 10.f), Point2f(0.f, 10.f)}};
+    vector<vector<Point2d>> detected_double = {{Point2d(0., 0.), Point2d(10., 0.), Point2d(10., 10.), Point2d(0., 10.)}};
+    vector<vector<Point2d>> detected_int = {{Point(0, 0), Point(10, 0), Point2d(10, 10), Point2d(0, 10)}};
```

Comment:
The same test type `Point2d` and `Point2d`. Should it be `2i` in the second case?

Reply:
fixed:
`vector<vector<Point2i>> detected_int = {{Point(0, 0), Point(10, 0), Point(10, 10), Point(0, 10)}};`